server/autolink/lib_jira_test.go

```diff
@@ -118,6 +118,42 @@ var jiraTests = []linkTest{
 		"Welcome https://mattermost.atlassian.net/browse/MM-12345",
 		"Welcome [MM-12345](https://mattermost.atlassian.net/browse/MM-12345)",
 	},
+	{
+		"Jump To Comment With Jira Cloud",
+		autolink.Autolink{
+			Pattern:  "(https://mmtest.atlassian.net/browse/)(DP)(-)(?P<jira_id>\\d+)[?](focusedCommentId)(=)(?P<comment_id>\\d+)",
+			Template: "[DP-${jira_id} With Focused Comment($comment_id)](https://mmtest.atlassian.net/browse/DP-${jira_id}?focusedCommentId=$comment_id)",
+		},
+		"https://mmtest.atlassian.net/browse/DP-454?focusedCommentId=11347",
```

Comment:
Can we write a test that doesn't have `focusedCommentId` in the URL, to test that the regular issue link case works?

Reply:
https://github.com/mattermost/mattermost-plugin-autolink/pull/175#discussion_r757184167 should cover it as well I think.